sysdig/secure/teams.go

```diff
@@ -0,0 +1,136 @@
+package secure
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+func (client *sysdigSecureClient) getUserIdbyEmail(userRoles []UserRoles) ([]UserRoles, error) {
```

Comment:
Sysdig API(/api/teams) requires User ID as identifier(not email address).
But I use email address as identifier on `sysdig_secure_team` resource.

This func gets User ID using Sysdig API(/api/users/light).

Reply:
Perfect! We can use it to implement a Data Source for Users :slightly_smiling_face: